rewrite-maven/src/main/java/org/openrewrite/maven/tree/ResolvedPom.java

```diff
@@ -1179,17 +1194,20 @@ private Scope getDependencyScope(Dependency d2, ResolvedPom containingPom) {
         Scope scopeInContainingPom;
         //noinspection ConstantConditions
         if (d2.getScope() != null) {
+            // TODO: Unsure if should be resolving scope similarly to classifier below
             scopeInContainingPom = Scope.fromName(getValue(d2.getScope()));
         } else {
+            // TODO: Unsure if should be resolving group id / artifact id / version similarly to classifier
```

Comment:
I think it would indeed make sense to have similar handling for the other GAV coordinates, as they might similarly be using properties (although perhaps less likely so).